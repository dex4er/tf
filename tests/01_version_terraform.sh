#!/usr/bin/env bash

set -euo pipefail

cd $(dirname $0)
test=$(basename $0 .sh)

export TERRAFORM_PATH=${test##*_}

rm -rf tmp/$test
mkdir -p tmp/$test
cp ../demo.tf tmp/$test
pushd tmp/$test >/dev/null

{
  ../../../tf version
} | ../../sanitize.sh 2>&1 >> tf.out

diff -u ../../$test.out tf.out

popd >/dev/null

rm -rf tmp/$test
