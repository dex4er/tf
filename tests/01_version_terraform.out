tf vX.X.X
Terraform vX.X.X
on XXX
